### PR TITLE
feat: add ci-helm

### PR DESCRIPTION
**Pull request by fredleger**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      - id: should-build-image
+        uses: ./.github/actions/should-build-image
+        with:
+          base-sha: ${{ steps.get-latest-tag-sha.outputs.sha }}
+          build-context: images/${{ matrix.image }}
+
+      - id: release
+        if: steps.should-build-image.outputs.should-build-image == 'true'
+        # FIXME: wait for PR to be merged https://github.com/release-drafter/release-drafter/pull/1255
+        uses: neilime/release-drafter@fix/strip-prefix-before-comparing-version
```
Comment (neilime) [inline .github/workflows/release.yml:52]: This fixme can be fixed as a new version of release drafter is available

**Review by neilime — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (fredleger) [reply .github/workflows/release.yml:52]: Good just as a note this is cut and paste from ac repo so it might be applicable here too

Comment (neilime) [reply .github/workflows/release.yml:52]: Where did you find it?

File: .github/actions/should-build-image/action.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - id: changed-files
+      uses: tj-actions/changed-files@v35.7.12
+      with:
+        base_sha: ${{ inputs.base-sha }}
+        since_last_remote_commit: ${{ github.event_name == 'push' }}
+        files: |
+          ./${{ inputs.build-context }}/*
+          .github/actions/should-build-image/*
+          .github/workflows/__shared-ci.yml
+          .github/workflows/__shared-get-available-images-matrix.yml
```
Comment (neilime) [inline .github/actions/should-build-image/action.yml:37]: I've made some improvements on this action
```suggestion
    - if: ${{ github.event_name == 'push' }}
      uses: actions/checkout@v3
      with:
        fetch-depth: 2
    - if: ${{ github.event_name != 'push' }}
      uses: actions/checkout@v3
      with:
        fetch-depth: 0

    - id: changed-files
      uses: tj-actions/changed-files@v37.4.0
      with:
        base_sha: ${{ inputs.base-sha }}
        since_last_remote_commit: ${{ github.event_name == 'push' }}
        files: |
          ${{ inputs.build-context }}/
          .github/actions/should-build-image/
          .github/workflows/__shared-ci.yml
          .github/workflows/__shared-get-available-images-matrix.yml
```

File: .github/workflows/release.yml (hunk 1 above)
Comment (neilime) [inline .github/workflows/release.yml:52]:
```suggestion
        uses: release-drafter/release-drafter@v5.24.0
```

**Review by neilime — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (fredleger) [reply .github/workflows/release.yml]: Can't find it anymore maybe my main branch was outdated when i copy/paste to here

**Pull request merged**